app/src/main/java/com/mediaan/mymediaan/model/Interests.kt

```diff
@@ -0,0 +1,28 @@
+package com.mediaan.mymediaan.model
+
+enum class Interest {
+    MUSIC,
+    SPORTS,
+    HIKING,
+    DOGS,
+    CATS,
+    READING,
+    TRAVELING,
+    COOKING,
+    GAMING,
+    PHOTOGRAPHY,
+    PAINTING,
+    GARDENING,
+    WRITING,
+    FISHING,
+    BIKING,
+    SWIMMING,
+    YOGA,
+    DANCING,
+    TECHNOLOGY,
+    FITNESS;
```

Comment:
I would be confused to see both "SPORTS", "FITNESS", "BIKING" and "SWIMMING". Maybe let's leave one - "SPORTS".

Reply:
...I may or may not have asked ChatGPT to generate this list. Updating it.